loader/log.c

```diff
@@ -149,88 +150,82 @@ void loader_log(const struct loader_instance *inst, VkFlags msg_type, int32_t ms
         util_SubmitDebugUtilsMessageEXT(inst, severity, type, &callback_data);
     }
 
-    if (!(msg_type & g_loader_debug)) {
+    uint32_t filtered_msg_type = (msg_type & g_loader_debug);
+    if (0 == filtered_msg_type) {
         return;
     }
 
     cmd_line_msg[0] = '\0';
     cmd_line_size -= 1;
```

Comment:
Should we not modify `cmd_line_size` anymore?

Reply:
I could get rid of it, but this just makes sure there's space for a null terminator.  I could update num_used instead.